### PR TITLE
Project Movies 🍿

### PR DESCRIPTION
**Pull request by destinyleia**
Link to my deployed project:
moviemakers.netlify.app

My reflections on how this week's project turned out:
Read my readme!

Things I'd like to have clarified or explained in more detail:
I like to code along in class and then repeating it alot - so would love more of that in class! And more explaining about the code-language ! :D

### CODE REVIEW

File: code/src/index.css [hunk 1]
```diff
@@ -1,13 +1,154 @@
...
+  position: absolute;
+}
+
+.movie-section:hover .movies-background-img { 
+  opacity: 0.3;
+  z-index: 1;
+  background: rgba(0, 0, 0, 0.75);
+}
+
+
```
Comment (nathalienordstrom) [inline code/src/index.css:73]: nice solution for the hovering

**Review by nathalienordstrom — COMMENTED** [review]
Great job Destiny! Looks clean and nice!

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,22 @@
 import React from 'react'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
+
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply code/src/App.js:3]: That space is pretty important :

There is a convention which says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, `style.css` should be separated as well and located below local dependencies. Ideally, it should look like this :

```jsx
import React from 'react';
import ReactDOM from 'react-dom';

import MyComponent1 from './components/MyComponent1';
import MyComponent2 from './components/MyComponent2';
import MyComponent3 from './components/MyComponent3';

import './style.css';
```

So, final order is :

1. Outer dependencies
2. Local dependencies
3. Style file(s)

File: code/src/components/MovieList.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+  return (
+    <>
+      <section className="movie-list">
+        {movies.map((movie) => (
+          <div
+            className="movie-section"
+            key={movie.id}>
+            <Link
+              key={movie.id}
```
Comment (EmOlsz) [inline code/src/components/MovieList.js:25]: No need for `key` here, we already added it in line 23. It should only be added to the root element (element which wraps all other) of loop

**Pull request closed without merging**